targeted/cve-2021-22214.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Contributed by Astra Security (https://www.getastra.com/)
+ * @author Prince Mendiratta <prince.mendiratta@getastra.com>
+ */
+
+var pluginid = 1204616;
```

Comment:
This isn't [registered](https://github.com/zaproxy/zaproxy/blob/develop/docs/scanners.md) and can be left out.

Reply:
Have sent PR to register for the plugin ID.